### DISCUSSION TITLE
关于资源分享

### DISCUSSION THREAD

**Discussion by movejian** [agalwood/Motrix#1221] [Ideas]
我在资源分享里面看到其他客户端分享给我们的资源，唯独迅雷在线 (Xunlei) 只有上传数据，没有下载数据，我们是不是在某个地方设置个屏蔽某些客户端或者某些IP。
迅雷违背了BT协议，只索取，不付出。迅雷不具备有分享精神，我们也应该拒绝分享资源给他，没有奉献精神。

**Comment by Saboteur18** [comment]
你可以尝试使用qbittorrent，那个可以开反吸血

**Reply by movejian** [reply]
你说的对，这样我们可以不需要用这个motrix了。这个东西，缺陷还是很多，虽然集于一身，但是要配合各种下载器
